### ISSUE TITLE
[FEATURE] Add "Deep sleep" with BTN1 enable/disable in Settings

### ISSUE THREAD

**Issue by ErikS-web** [kamilsss655/ESPRI#37]
### Is your feature request related to a problem? Please describe.
No

### Describe the solution you'd like
When the touchpad  marked as `BTN1` on the ESPRi-PCB the program flashed twice the Green LED and than goes to standby.

This can be seen in console-logging.
> I (12777) HW/BUTTON: Button 1 was pressed.
> I (12877) SYSTEM: Deep sleep

Until BTN1 is pressed again (or reset from device).
> rst:0x5 (DEEPSLEEP_RESET),boot:0x13 (SPI_FAST_FLASH_BOOT)

![Touch-pad-1-marked](https://github.com/kamilsss655/ESPRI/assets/148579604/a4c3648a-ec77-49e6-a465-d20d90942319)

### Describe alternatives you've considered
Not everyone will prefer this and may even touch it unconsciously.
I suggest making this `BTN-1 standby function selectable in Setting`

### Additional context
Add any other context, screenshots or URL to the website about the feature request here.

**Comment by kamilsss655** [comment]
Yup, ideally there should be actions system, where BTN 1 has user assignable action/function.

Perhaps good first issue for new developers.
